sw/device/lib/testing/mock_csr_test.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/mock_csr.h"
+
+#include "gtest/gtest.h"
+#include "sw/device/lib/base/csr.h"
+
+namespace {
+using ::testing::Return;
```

Comment:
Is this necessary for the code to compile?

Reply:
I don't think so, I think this is left over from an earlier iteration. I'll remove it.

Reply:
Fixed